### ISSUE TITLE
Lasso tool for selection in addition to box

### ISSUE THREAD

**Issue by fmaussion** [OGGM/world-glacier-explorer#8]
@linznicholson says: _the square selection box makes it tricky to pick out specific mountain ranges for example_

**Comment by zschirmeister** [comment]
It should be possible to easily add 'lasso_select': when I understand it right they discuss it here: https://github.com/holoviz/holoviews/issues/4359 it should be possible from holoviews v1.13.3 onwards. But I am unable to set up my environment in the right way at the moment to test it.

**Comment by fmaussion** [comment]
Yes - how much do we want that? If we want it hard we can try to spend the time on it
